tinywallet/tinywallet/screens.py

```diff
@@ -66,13 +68,14 @@ class TinyDialog(QtWidgets.QFrame):
     successSig = QtCore.pyqtSignal(str)
     errorSig = QtCore.pyqtSignal(str)
 
-    def __init__(self, app):
+    def __init__(self, qApp):
         """
         Args:
             app (TinyDecred): The TinyDecred application instance.
```

Comment:
Update `qApp` description. Also, the app should be called `TinyWallet`, not `TinyDecred`, right?